### DISCUSSION TITLE
Should I be still using this package?

### DISCUSSION THREAD

**Discussion by 16ntu1120** [react-native-async-storage/async-storage#571] [Q&A] [answered]
I found this package deprecated should I be still using it? Need suggestions.

**Comment by krizzu** [comment, marked as answer]
Hey,

`AsyncStorage` is marked as `deprecated` in `React Native`, because it was extracted to a separate package (this repo). So ideally, if you used previously3 AsyncStorage from `react-native` package, you should use this one instead

**Comment by 16ntu1120** [comment]
@Krizzu okay thanks man
